src/impl/tpls/KokkosBlas3_gemm_tpl_spec_decl.hpp

```diff
@@ -389,11 +510,23 @@ struct GEMM< \
       transb = CUBLAS_OP_T; \
     else \
       transb = CUBLAS_OP_C; \
-    KokkosBlas::Impl::CudaBlasSingleton & s = KokkosBlas::Impl::CudaBlasSingleton::singleton(); \
-    if(!A_is_lr && !B_is_lr && !C_is_lr ) \
-      cublasDgemm(s.handle, transa, transb, M, N, K, &alpha, A.data(), LDA, B.data(), LDB, &beta, C.data(), LDC); \
-    if(A_is_lr && B_is_lr && C_is_lr ) \
-      cublasDgemm(s.handle, transb, transa, N, M, K, &alpha, B.data(), LDB, A.data(), LDA, &beta, C.data(), LDC); \
+    \
```

Comment:
Lines 496-498 might issue a warning, because the strides might have unsigned type.

Reply:
Those lines existed before my PR. Maybe @srajama1 has some input?